src/client.js

```diff
@@ -469,6 +468,12 @@ export class StreamChat {
 
 	doAxiosRequest = async (type, url, data, params = {}) => {
 		await this.tokenManager.tokenReady();
+
+		if (this.rateLimitResetTime) {
+			await sleep(this.rateLimitResetTime * 1000 - new Date());
```

Comment:
might be better to add small noise so that all doesn't come at the same time

Reply:
noise?

Reply:
jitter/delay - some small random time

Reply:
Makes sense!! Will do :)